SWIG/types.i

```diff
@@ -59,7 +59,7 @@ typedef Real Probability;
 }
 #endif
 
-typedef std::size_t Size;
+typedef size_t Size;
```

Comment:
That's strange; `Size` was already used with this definition without problems.  What version of SWIG are you using, and what problems were you getting with the previous definition?

Reply:
SWIG Version 3.0.5

Compiled with clang++ [x86_64-apple-darwin14.1.0]

Configured options: +pcre

It actually does the code generation fine, but has an error on compile. this was the minimal fix

Anyway, thanks as always for your quick and detailed comments. I know its the holidays and wasn't expecting a response until January.

Reply:
I believe the error was due to me defining std::vector<Size>, something in swigs vector wrapper was having issue wit std::size_t